src/components/ExploreListing/explore-listing.styles.js

```diff
@@ -1,3 +1,169 @@
 import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
 
-export const Title = styled.h3``;
+export const MainHeader = styled.h2`
+  color: ${colors.mineshaft};
+  font-size: 1.37rem;
+  margin: 0 auto 1.1rem auto;
+  text-align: center;
+
+  ${mq.tablet`
+    font-size: 2rem;
+  `}
+`;
+
+export const SliderContainer = styled.div`
+  & .slick-slider {
```

Comment:
if we could remove the `&` from the styles as well, wherever it's used.
```suggestion
  .slick-slider {
```